app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-page-views.js

```diff
@@ -1,113 +1,114 @@
+          updated_at: this.stripTimeFrom(this.getMetaContent('updated-at')),
+          public_updated_at: this.stripTimeFrom(this.getMetaContent('public-updated-at')),
+          publishing_government: this.getMetaContent('publishing-government'),
+          political_status: this.getMetaContent('political-status'),
+          primary_publishing_organisation: this.getMetaContent('primary-publishing-organisation'),
+          organisations: this.getMetaContent('analytics:organisations'),
+          world_locations: this.getMetaContent('analytics:world-locations')
         }
-        window.dataLayer.push(data)
-      }
-    },
-
-    getLocation: function () {
-      return this.PIIRemover.stripPII(document.location.href)
-    },
-
-    getReferrer: function () {
-      return this.PIIRemover.stripPIIWithOverride(document.referrer, true, true)
-    },
-
-    getTitle: function () {
-      return this.PIIRemover.stripPII(document.title)
-    },
-
-    // window.httpStatusCode is set in the source of the error page in static
-    // https://github.com/alphagov/static/blob/main/app/views/root/_error_page.html.erb#L32
-    getStatusCode: function () {
-      if (window.httpStatusCode) {
-        return window.httpStatusCode.toString()
-      } else {
-        return '200'
-      }
-    },
-
-    getMetaContent: function (name) {
-      var tag = document.querySelector('meta[name="govuk:' + name + '"]')
-      if (tag) {
-        return tag.getAttribute('content')
-      } else {
-        return this.nullValue
-      }
-    },
-
-    getLanguage: function () {
-      var content = document.getElementById('content')
-      if (content) {
-        return content.getAttribute('lang') || this.nullValue
-      } else {
-        return this.nullValue
-      }
-    },
-
-    getHistory: function () {
-      var history = this.getMetaContent('content-has-history')
-      return (history === 'true') ? 'true' : 'false'
-    },
-
-    getWithDrawn: function () {
-      var withdrawn = this.getMetaContent('withdrawn')
-      return (withdrawn === 'withdrawn') ? 'true' : 'false'
-    },
-
-    // return only the date from given timestamps of the form
-    // 2022-03-28T19:11:00.000+00:00
-    stripTimeFrom: function (value) {
-      if (value !== null) {
-        return value.split('T')[0]
-      } else {
-        return this.nullValue
       }
+      window.dataLayer.push(data)
+    }
+  }
+
+  PageViewTracker.getLocation = function () {
+    return this.PIIRemover.stripPII(document.location.href)
+  }
+
+  PageViewTracker.getReferrer = function () {
+    return this.PIIRemover.stripPIIWithOverride(document.referrer, true, true)
+  }
+
+  PageViewTracker.getTitle = function () {
+    return this.PIIRemover.stripPII(document.title)
+  }
+
+  // window.httpStatusCode is set in the source of the error page in static
+  // https://github.com/alphagov/static/blob/main/app/views/root/_error_page.html.erb#L32
```

Comment:
Currently line 32 linked to here is `<div class="govuk-grid-row">`, which I don't think is what's wanted here. Could we link to a specific commit to avoid this kind of drift?

```suggestion
  // https://github.com/alphagov/static/blob/1c734451f2dd6fc0c7e80beccbdcbfa5aaffd0e4/app/views/root/_error_page.html.erb#L41-L43
```

Reply:
Good shout, have updated, thanks.